config/application.yml.example

```diff
@@ -179,7 +179,7 @@ development:
   piv_cac_verify_token_secret: 'ee7f20f44cdc2ba0c6830f70470d1d1d059e1279cdb58134db92b35947b1528ef5525ece5910cf4f2321ab989a618feea12ef95711dbc62b9601e8520a34ee12'
   piv_cac_service_url: 'https://localhost:8443/'
   piv_cac_verify_token_url: 'https://localhost:8443/'
-  pkcs11_lib: '/opt/cloudhsm/lib/libcloudhsm_pkcs11.so'
+  pkcs11_lib:
```

Comment:
was dropping '/opt/cloudhsm/lib/libcloudhsm_pkcs11.so' from the config deliberate?

Reply:
Looks like it's missing in the prod config.

Reply:
Hmm, yeah, lemme clean this up a little bit.

Reply:
Okay, I cleaned up the initializer to be a little bit smarter and put the value for this back into the config.